src/include/session.h

```diff
@@ -76,6 +76,11 @@ struct WT_COMPILER_TYPE_ALIGN(WT_CACHE_LINE_ALIGNMENT) __wt_session_impl {
 	WT_CURSOR_BACKUP *bkp_cursor;	/* Hot backup cursor */
 	WT_COMPACT	 *compact;	/* Compact state */
 
+	/*
+	 * Lookaside file cursor, sweep thread and eviction worker threads only.
+	 */
+	WT_CURSOR	*las_cursor;	/* Lookaside file cursor */
+
 	WT_DATA_HANDLE *meta_dhandle;	/* Metadata file */
```

Comment:
So, instead of instantiating a new metadata cursor for each operation, have a shared (locked) metadata cursor? Given the locking we have to do to instantiate the metadata cursor, I expect it would be faster for simple (insert/update/remove) operations.

Sure, if you will confirm that's the plan, I'll code it up.

There are some potentially longer-running operations where we might want to continue to instantiate a new metadata cursor: maybe hot backup, meta-btree-apply.

Should we used the shared cursor for those operations, or instantiate a new one? If we instantiate a new one, should we keep the meta_dhandle around to make that faster, or not bother?